console/src/assets/i18n/bg.json

```diff
@@ -1384,7 +1384,8 @@
         "sv": "Svenska",
         "id": "Bahasa Indonesia",
         "hu": "Magyar",
-        "ko": "한국어"
+        "ko": "한국어",
```

Comment:
Translation for `POLICY.LOGIN_TEXTS.LANGUAGES.tr` is missing in all translation files in this folder

Reply:
@hbasria this comment wasn't resolved. This translation needs to be added here:

https://github.com/zitadel/zitadel/blob/b60d316ea9b16a2507e02c85d16aba66e7457672/console/src/assets/i18n/bg.json#L1610-L1626